modules/strelka/1.0/strelka.smk

```diff
@@ -52,21 +52,25 @@ rule _strelka_input_bam:
         op.relative_symlink(input.bai, output.bai)
 
 
-rule _strelka_input_vcf:
-    input:
-        vcf = CFG["inputs"]["candidate_small_indels_vcf"],
-        tbi = CFG["inputs"]["candidate_small_indels_tbi"]
+rule _strelka_dummy_vcf:
     output:
-        vcf = CFG["dirs"]["inputs"] + "{seq_type}--{genome_build}/vcf/{tumour_id}--{normal_id}--{pair_status}.candidateSmallIndels.vcf.gz",
-        tbi = CFG["dirs"]["inputs"] + "{seq_type}--{genome_build}/vcf/{tumour_id}--{normal_id}--{pair_status}.candidateSmallIndels.vcf.gz.tbi"
-    run:
-        op.relative_symlink(input.vcf, output.vcf)
-        op.relative_symlink(input.tbi, output.tbi)
+        touch(CFG["dirs"]["inputs"] + "{seq_type}--{genome_build}/vcf/{tumour_id}--{normal_id}--{pair_status}.dummy.tbi")
 
 
-rule _strelka_dummy_vcf:
-    output:
-        touch(CFG["dirs"]["inputs"] + "{seq_type}--{genome_build}/vcf/{tumour_id}--{normal_id}--{pair_status}.dummy")
+if not CFG["inputs"]["candidate_small_indels_tbi"] and CFG["inputs"]["candidate_small_indels_vcf"]:
```

Comment:
Wait, does this work?? I never considered having a rule in an if statement.

Reply:
It works for the null statements and I'm just about to test on the other condition.

Reply:
so it doesn't work because I think they try to figure the DAG before the if statements?

Reply:
i think its working now